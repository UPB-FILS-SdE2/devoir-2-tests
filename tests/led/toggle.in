led toggle 0 0
led toggle 0 0
led toggle 4 2
exit