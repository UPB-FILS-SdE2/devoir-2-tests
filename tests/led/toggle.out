cmd: Set pixel 0 0 to 9
cmd: Set pixel 0 0 to 0
cmd: Set pixel 4 2 to 9
cmd: 